File: backend/components/metiers/administration_publique/legacy/fallback/samples/sample_fallback_legacy.test.js
// sample_fallback_legacy.test.js – Test JS exemple fallback legacy
const { sampleFallbackLegacy } = require('./sample_fallback_legacy');
test('sampleFallbackLegacy fonctionne', () => {
  expect(sampleFallbackLegacy({ a: 1 })).toEqual({ a: 1, fallback: true });
  expect(sampleFallbackLegacy()).toEqual({ fallback: true, empty: true });
});

File: backend/components/metiers/administration_publique/legacy/fallback/samples/sample_fallback_legacy.test.py
# sample_fallback_legacy.test.py – Test Python exemple fallback legacy
from .sample_fallback_legacy import sample_fallback_legacy

def test_sample_fallback_legacy_ok():
    assert sample_fallback_legacy({'a': 1}) == {'a': 1, 'fallback': True}
    assert sample_fallback_legacy(None) == {'fallback': True, 'empty': True}
